README.adoc

```diff
@@ -8,14 +8,13 @@
 This guide provides a sampling of how {spring-boot}[Spring Boot] helps you accelerate
 application development. As you read more Spring Getting Started guides, you will see more
 use cases for Spring Boot. This guide is meant to give you a quick taste of Spring Boot.
-If you want to create your own Spring Boot-based project, visit
-https://start.spring.io/[Spring Initializr], fill in your project details, pick your
-options, and download a bundled up project as a zip file.
```

Comment:
I think it's still worth keeping a link to start.spring.io

Reply:
Definitely. Our mantra at conferences and elsewhere is "start dot spring dot io". We can't steer away from that without changing our strategy.

Reply:
In the spring-guides repo I'd meant to move this, not delete it. So in that repo this content is back.